### PR TITLE
SearxNG brand

### PR DESCRIPTION
**Pull request by return42**
## What does this PR do?

Fix links and documentation to point to searxng .... fore details see commits.

### CODE REVIEW

File: docs/admin/settings.rst [hunk 1]
```diff
@@ -43,15 +43,16 @@ Global Settings
...
+       instance_name : "searxng" # displayed name
+       contact_url: False # mailto:contact@example.com
+
+   brand:
+       git_url: https://github.com/saerxng/saerxng
        git_branch: master
-       issue_url: https://github.com/searx/searx/issues
-       docs_url: https://searx.github.io/searx
+       issue_url: https://github.com/saerxng/saerxng/issues
+       docs_url: https://saerxng/saerxng.github.io/saerxng
```
Comment (dalf) [inline docs/admin/settings.rst]: small typo on git_url / issue_url / docs_url & wiki_url (below)

File: searx/templates/__common__/about.html [hunk 2]
```diff
@@ -80,9 +79,9 @@ <h2>Where to find anonymous usage statistics of this instance ?</h2>
   <h2>How can I make it my own?</h2>
 
   <p>
-    Searx appreciates your concern regarding logs, so take the
-    code from the <a href="https://github.com/searx/searx">original searx project</a> and
-    run it yourself!
+    Searx appreciates your concern regarding logs, so take the code from
```
Comment (dalf) [inline searx/templates/__common__/about.html]: should searx be replaced by searxng  ?

Comment (return42) [reply searx/templates/__common__/about.html]: I replaced searx by SearXNG in the complete _about_ side. https://github.com/searxng/searxng/pull/25/commits/abfabebd9173086d815c19e34212f1f24676c5ee#diff-7dc09c2bec99a1049f972f8d2d5f19db05b8ee5064452036c232f995eac3a603

File: docs/admin/settings.rst (hunk 1 above)
Comment (return42) [reply docs/admin/settings.rst]: Thanks / done.

File: CHANGELOG.rst [hunk 3]
```diff
@@ -180,7 +180,7 @@ New settings.yml
 - ``ui.advanced_search`` - add preference for displaying advanced settings ( #2327 )
 - ``server.method: "POST"`` - Make default query submission method configurable ( #2130 )
 - ``server.default_http_headers`` - add default http headers ( #2295 )
-- ``engines.*.proxies`` - Using proxy only for specific engines ( #1827 #2319 ), see https://searx.github.io/searx/dev/engine_overview.html#settings-yml
+- ``engines.*.proxies`` - Using proxy only for specific engines ( #1827 #2319 ), see https://sarxng.github.io/searxng/dev/engine_overview.html#settings-yml
```
Comment (dalf) [inline CHANGELOG.rst]: A last typo: sarxng -> searxng (and below lines 299 and 300)

Comment (return42) [reply CHANGELOG.rst]: Oops .. sorry / fixed now.

**Pull request merged**